lightning_integration_test.py

```diff
@@ -12,32 +11,31 @@
 
 # Test echoing using httpie
 # Try a background fork/thread
-server_process = subprocess.Popen(['./lightning', 'simple_config'])
+server_process = Popen(['./lightning', 'simple_config'])
 
 # Spawn a shell process to act as hanging http request
 # This tests multithreading
-telnet_request_command = "telnet localhost 2020"
-telnet_request_proc = subprocess.Popen(telnet_request_command, stdout=subprocess.PIPE, shell=True)
+telnet_request_command = "telnet localhost 8080"
+telnet_request_proc = Popen(telnet_request_command, stdout=PIPE, shell=True)
 
 # TODO: Have intermediate logging throughout
 # TODO: Use Python unit test frameworks + logging libraries
 print('DEBUG: Lightning server started!')
 
-
 # as the telnet is hanging, we will perform an echo request, which should still be 
 # handled by a separate handler in a new thread; it will still succeed and return 
 # a response instantly.
 
 expected_response = b"""GET /echo HTTP/1.1\r\nHost: localhost:8080\r\n\
 Accept-Encoding: gzip, deflate, compress\r\n\
 Accept: */*\r\nUser-Agent: HTTPie/0.8.0\r\n\r\n"""
-actual_response = run(['http', 'localhost:8080/echo'], stdout=subprocess.PIPE)
+actual_response = Popen(['http', 'localhost:8080/echo'], stdout=PIPE)
 
-if (actual_response.returncode != 0):
-    print('FAILED: httpie encountered an error')
-    ec = 1;
+# if (actual_response.returncode != 0):
```

Comment:
`returncode` is a Python 3.6 feature. Can we remove these lines altogether? We'll still have them in the git history.